draft-ietf-webpush-protocol.xml

```diff
@@ -305,7 +305,22 @@
         </t>
       </section>
     </section>
-
+    <section title="Connecting to the Push Service">
+      <t>
+        The push service shares the same default port number (443/TCP) with HTTPS, but MAY
+        also advertise the IANA allocated TCP port 1001 using HTTP alternative services

```

Comment:
I've had experience with IANA not liking this, but if it's in 6335, then that's OK.  Just identifying that this is a system port would be fine as well.